### PR TITLE
Add more unit tests for InodeLockList

### PR DESCRIPTION
**Pull request by cheng-chang**

### CODE REVIEW

**Comment by cheng-chang** [comment]
jenkins, test this please

File: core/server/master/src/test/java/alluxio/master/file/meta/SimpleInodeLockListTest.java [hunk 1]
```diff
@@ -212,4 +212,105 @@ public void doubleWriteLock() {
...
   }
+
+  @Test
+  public void readAfterWrite() {
+    mLockList.lockRootEdge(LockMode.READ);
+    assertEquals(LockMode.READ, mLockList.getLockMode());
+    mLockList.lockInode(mRootDir, LockMode.WRITE);
+    assertEquals(LockMode.WRITE, mLockList.getLockMode());
+    mLockList.lockEdge(mRootDir, mDirA.getName(), LockMode.READ);
+    assertEquals(LockMode.WRITE, mLockList.getLockMode());
```
Comment (ggezer) [inline core/server/master/src/test/java/alluxio/master/file/meta/SimpleInodeLockListTest.java]: You are now asserting for this on https://github.com/Alluxio/alluxio/pull/9092

Comment (cheng-chang) [reply core/server/master/src/test/java/alluxio/master/file/meta/SimpleInodeLockListTest.java]: I reverted to original behavior in that PR.

**Comment by cheng-chang** [comment]
@gpang PTAL, thanks.

**Comment by cheng-chang** [comment]
alluxio-bot, merge this please

**Pull request merged**